src/pint/bayesian.py

```diff
@@ -0,0 +1,206 @@
+from copy import deepcopy

```

Comment:
I think if you include a top-level docstring, it will show up at https://nanograv-pint--1407.org.readthedocs.build/en/1407/_autosummary/pint.html in the module description